apps/codeforafrica/src/lib/api.netlify-cms/getOurPartners.js

```diff
@@ -9,7 +9,11 @@ const pageDir = join(process.cwd(), "content/pages");
 
 export default function geOurPartners(page = "index") {
   const {
-    "our-partners": { title: originalTitle, "partners-list": partnersIds },
+    "our-partners": {
+      title: originalTitle,
+      "partners-list": partnersIds,
+      action,
```

Comment:
```suggestion
      action = null,
```

We only need to initialize `action` to `null` here in case it's `undefined` for about page.